### PR TITLE
WIP: 0.21.1: Updating chainparams and adding release notes

### PR DESCRIPTION
**Pull request by DavidBurkett**

### CODE REVIEW

File: doc/litecoin-release-notes/release-notes-0.21.1.md [hunk 1]
```diff
@@ -0,0 +1,1200 @@
+0.21.1 Release Notes
+====================
+
+Litecoin Core version 0.21.1 is now available from:
+
+  <https://download.litecoin.org/litecoin-0.21.0/>
```
Comment (litediss) [inline doc/litecoin-release-notes/release-notes-0.21.1.md:6]: do you mean .../litecoin-0.21.1/ ?

File: doc/litecoin-release-notes/release-notes-0.21.1.md [hunk 2]
```diff
@@ -0,0 +1,1200 @@
...
+as frequently tested on them.
+
+From Litecoin Core 0.21.1 onwards, macOS versions earlier than 10.12 are no
+longer supported. Additionally, Litecoin Core does not yet change appearance
+when macOS "dark mode" is activated.
+
+The node's known peers are persisted to disk in a file called `peers.dat`. The
+format of this file has been changed in a backwards-incompatible way in order to
+accommodate the storage of Tor v3 and other BIP155 addresses. This means that if
+the file is modified by 0.21.0 or newer then older versions will not be able to
```
Comment (litediss) [inline doc/litecoin-release-notes/release-notes-0.21.1.md:50]: 0.21.0 or 0.21.1 ?

File: doc/litecoin-release-notes/release-notes-0.21.1.md [hunk 3]
```diff
@@ -0,0 +1,1200 @@
...
+  configuration options and in the `setban` RPC. (#19628)
+
+- The `-blocksonly` setting now completely disables fee estimation. (#18766)
+
+Changes to Wallet or GUI related settings can be found in the GUI or Wallet section below.
+
+Tools and Utilities
+-------------------
+
+- A new `bitcoin-cli -netinfo` command provides a network peer connections
```
Comment (litediss) [inline doc/litecoin-release-notes/release-notes-0.21.1.md:237]: s/bitcoin/litecoin/g

File: doc/litecoin-release-notes/release-notes-0.21.1.md [hunk 4]
```diff
@@ -0,0 +1,1200 @@
...
+-----
+
+- The BIP 325 default signet can be enabled by the `-chain=signet` or `-signet`
+  setting. The settings `-signetchallenge` and `-signetseednode` allow
+  enabling a custom signet.
+
+- The `generateblock` RPC allows testers using regtest mode to
+  generate blocks that consist of a custom set of transactions. (#17693)
+
+0.21.0 change log
```
Comment (litediss) [inline doc/litecoin-release-notes/release-notes-0.21.1.md:542]: 0.21.0 + 0.21.1?

File: doc/litecoin-release-notes/release-notes-0.21.1.md [hunk 5]
```diff
@@ -0,0 +1,1200 @@
...
+- #19050 Add warning for rest interface limitation (fjahr)
+- #19390 doc/REST-interface: Remove stale info (luke-jr)
+- #19344 docs: update testgen usage example (Bushstar)
+
+Credits
+=======
+
+Thanks to everyone who directly contributed to this release:
+
+- [The Bitcoin Core Developers](https://github.com/bitcoin/bitcoin/tree/master/doc/release-notes)
```
Comment (litediss) [inline doc/litecoin-release-notes/release-notes-0.21.1.md:1198]: Litecoin Developers?

**Review by litediss — COMMENTED** [review]
First contribution, sorry to bother on a WIP file.

File: doc/litecoin-release-notes/release-notes-0.21.1.md (hunk 5 above)
Comment (DavidBurkett) [reply doc/litecoin-release-notes/release-notes-0.21.1.md:1198]: No, this is saying the bitcoin developers plus the few litecoin devs underneath.

File: doc/litecoin-release-notes/release-notes-0.21.1.md (hunk 4 above)
Comment (DavidBurkett) [reply doc/litecoin-release-notes/release-notes-0.21.1.md:542]: Can change it to just 0.21.1

**Comment by zawy12** [comment]
Maybe change 2 weeks to 3.5 days?  Some day this old difficulty algo might get the chain stuck from oscillations. LTC might be the only coin that has been big enough to use it and not get stuck. No plans to change it?

https://github.com/litecoin-project/litecoin/blob/508dfae2a878af97ab9a28e50233d8a2414433fb/src/chainparams.cpp#L282

**Pull request merged**